serenata_toolbox/federal_senate/federal_senate_dataset.py

```diff
@@ -0,0 +1,83 @@
+import os.path
+from urllib.request import urlretrieve
+import numpy as np
+import pandas as pd
+
+class FederalSenateDataset:
+    URL = 'http://www.senado.gov.br/transparencia/LAI/verba/{}.csv'
+
+    def __init__(self, path):
+        self.path = path
+
+    def fetch(self):
+        urls = [self.URL.format(year) for year in range(2008, 2018)]
+        filename_from_url = lambda url: 'federal-senate-{}'.format(url.split('/')[-1])
+        filenames = map(filename_from_url, urls)
+
+        for url, filename in zip(urls, filenames):
+            csv_file_path = os.path.join(self.path, filename)
+            urlretrieve(url, csv_file_path)
+
+    def translate(self):
+        filenames = ['federal-senate-{}.csv'.format(year) for year in range(2008, 2018)]
```

Comment:
Oops… this is the 2nd time I see this `range(2008, 20180)`! Let's make it a constant (at least a class constant like `URL`).